improver/cli/combine.py

```diff
@@ -62,6 +63,8 @@ def process(
             period).  If True, uses the mid-point.
         check_metadata (bool):
             If True, warn on metadata mismatch between inputs.
+        broadcast_to_coord (str):
+            Name of coord to broadcast one of the input cubes to prior to combining - must exist on the other input cube.
```

Comment:
This explanation assumes there are only two input cubes.  Is that an underlying assumption of the design?  (I'll suggest different things depending on the answer here.)

Reply:
Hmm...  Cube combiner can take any number of cubes. The multiply case currently has no use-case for more than two cubes although I wrote the code to cope with three or more so long as the first cube is the one from which all the meta-data should be inherited (and also the one with the full coordinates on it as they'll all be in the output cube).

Reply:
I've updated the help-string to indicate that many cubes are acceptable.